### PR TITLE
Kill command fix

### PR DESCRIPTION
**Pull request by immolo**

### CODE REVIEW

**Comment by dotaxis** [comment]
Also I don't want a clean exit. Please use `killall -9` so we don't have to wait 5 minutes for Steam to decide it's ready to close.

**Comment by immolo** [comment]
> Also I don't want a clean exit. Please use `killall -9` so we don't have to wait 5 minutes for Steam to decide it's ready to close.
> 
> And if we are using killall instead of pkill please change line 109 as well. It should work by adding the extension to the search string:
> 
> ```
> killall -9 "FF7_Launcher.exe"
> ```

This change is now complete.

**Comment by dotaxis** [comment]
Still waiting on that readme change.

Also if you can change the commit messages to reflect the changes I've asked for so I can do a merge commit, that'd be great. If not I can just squash and merge.

Thanks for your help on this.

**Comment by immolo** [comment]
Cleaned up the commit messages and fixed README to match your needs. This should be everything now but feel free to ask if you need something else changed.

**Review by dotaxis — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # 7thDeck
-Installer &amp; launcher for 7th Heaven on SteamOS and other Linux distros using KDE
+Installer &amp; launcher for 7th Heaven on SteamOS and other Linux distros
 
+This has been tested on KDE and MATE desktop environments however others should work
```
[the remark this answers is not in the document]
Comment (dotaxis) [reply README.md]: I forgot to publish this review before so I figured I'd do it now so I don't look crazy talking about readme changes I requested

**Review by dotaxis — APPROVED** [review]

File: install.sh [hunk 2]
```diff
@@ -106,7 +106,7 @@ echo "Sign into the Steam account that owns FF7 if prompted."
 nohup steam steam://rungameid/39140 &> /dev/null &
 echo "Waiting for Steam..."
 while ! pgrep "FF7_Launcher" > /dev/null; do sleep 1; done
-pkill -9 "FF7_Launcher"
+killall -9 "FF7_Launcher"
```
Comment (dotaxis) [inline install.sh:109]: This didn't work on my machine unless I added `.exe` to the search string

**Review by dotaxis — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (immolo) [reply install.sh]: > This didn't work on my machine unless I added `.exe` to the search string

Oops, you are right, I took your word at face value last time without testing.

Comment (dotaxis) [reply install.sh]: Not sure what you mean by this, I mentioned needing the extension in my first reply.

Comment (immolo) [reply install.sh]: I can't read it seem :)

**Review by dotaxis — APPROVED** [review]

**Comment by dotaxis** [comment]
Thank you!

**Pull request merged**